### PR TITLE
CaseStudyMarkdown

### PR DESCRIPTION
**Pull request by saritab07**
CaseStudy_markdown_version1

Added a Markdown file for the CaseStudy which illustrates the scenario of "Accelerating ML Model Development and Deployment on the Edge with the Machine Learning Model Factory"

### CODE REVIEW

File: CaseStudy.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+In conclusion, this technical case study highlights the significant challenges faced by the customer across various facets of ml model deployment and operations. These encompassed obstacles from slow model generation to challenging deployment processes and high operational cost.
+The solution implemented for the customer has not only streamlined their ML model generation and deployment processes but have also ushered in a structured, scalable approach to machine learning operations at scale. This meets the customer's objective of building ML models with speed and agility while simultaneously enhancing cost-effectiveness and reducing operational complexities.
+
+## References
+
+-	[MLOps Model Factory Accelerator](https://github.com/microsoft/mlops-model-factory-accelerator)
+-	[MLOps Accelerator for Edge](https://github.com/microsoft/mlops-accelerator-for-edge)
+
+
+## Contact
```
Comment (ritesh-modi) [inline CaseStudy.md:158]: Can we remove the contact details?

File: CaseStudy.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# Case Study – “Accelerating ML Model Development and Deployment on the edge with the Machine Learning Model Factory”
```
Comment (ritesh-modi) [inline CaseStudy.md:1]: Case Study – “Accelerating ML Model Development and Deployment on the edge using MLOps Model Factory Accelerator

File: CaseStudy.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+     * [Data Collection on the Edge Design](data_collection_on_the_edge_design)
+     * [Deployment Size Optimizations for Constrained Network](deployment_size_optimizations_for_constrained_network)
+        * [Orchestrated Inferencing](Orchestrated_Inferencing)
+        * [Docker image optimizations](Docker_image_optimizations)
+     * [Deployment Size Reduction Data](Deployment_Size_Reduction_Data)
+ * [Conclusion](conclusion)
+ * [References](references)
+
+<!-- You can include any other section that is pertinent to your problem -->
+### Document Control:
```
Comment (ritesh-modi) [inline CaseStudy.md:24]: We can remove this section

File: CaseStudy.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+|     Document Version Details             |     Author(s)              |     Reviewer(s)    |     Date          |
+|------------------------------------------|----------------------------|--------------------|-------------------|
+|     Version 1.0 –   Detailed document    |     Sarita Behera          |     Vivek Soni     |     21st   Sep    |
+
+## Purpose
+This document depicts a real-world case study on how implementing a model factory enabled the customer, a large Telco company, to revolutionize its services by addressing challenges related to speed and agility in building and deploying machine learning models at scale.
```
Comment (ritesh-modi) [inline CaseStudy.md:30]: enabling a large Telco customer

File: CaseStudy.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+## Solution Overview
+To address these key challenges, the existing architecture has been revamped & reimagined by decoupling the model generation, from the business use case development and deployment process. This newer approach focuses on building a framework which can generate end-to-end machine learning use case with speed and agility.
+Technically, a ML use case can be defined as: A combination of single or multiple machine learning models coupled together by an orchestration mechanism to address a specific business problem.
+
+The conceptual design has 3 key components which function together to generate and deliver use cases to the target edge devices.
+
+![image](/docs/images/ConceptualDesign.jpg)
+
+- #### <ins>Machine Learning Model Factory:</ins>
```
Comment (ritesh-modi) [inline CaseStudy.md:72]: Can we add how many models we generated for the customer

File: CaseStudy.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+ ![image](/docs/images/MLModelFactory_Design.jpg)
+
+#### Key Features
+•	Supports generation of multiple ML Models
+•	MLOps pipeline for Data preparation, transformation, Model Training, evaluation, scoring and registration.
+•	Each ML Model is packaged in an independent Docker Image
+•	Model verification before storing the Docker image
+•	All Docker images are stored in Azure Container Registry
+•	Builds and deploys Smoke Test module on Edge device
+•	Based on Azure ML SDK v2 1.4 and IoT Edge runtime 1.4
```
Comment (ritesh-modi) [inline CaseStudy.md:101]: We can remove this point Based on Azure ML SDK v2 1.4 and IoT Edge runtime 1.4

File: CaseStudy.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
+# Case Study – “Accelerating ML Model Development and Deployment on the edge with the Machine Learning Model Factory”
+
+
+## Table of Contents
```
Comment (ritesh-modi) [inline CaseStudy.md:4]: None of the images have resolution clarity? Unable to read the text in the images

**Review by ritesh-modi — CHANGES_REQUESTED** [review]

File: CaseStudy.md (hunk 4 above)
Comment (saritab07) [reply CaseStudy.md]: Updated as advised.

File: CaseStudy.md (hunk 3 above)
Comment (saritab07) [reply CaseStudy.md]: Removed ToC and Document Control sections as advised

File: CaseStudy.md (hunk 2 above)
Comment (saritab07) [reply CaseStudy.md]: Updated the Title as advised,

File: CaseStudy.md (hunk 1 above)
Comment (saritab07) [reply CaseStudy.md]: Removed the contact details as advised,

File: CaseStudy.md (hunk 6 above)
Comment (saritab07) [reply CaseStudy.md]: Removed the bullet point

File: CaseStudy.md (hunk 7 above)
Comment (saritab07) [reply CaseStudy.md]: Added png format images with better resolution & clarity. Please check once

File: CaseStudy.md (hunk 5 above)
Comment (saritab07) [reply CaseStudy.md:42]: We generated one model for the customer. Shall we add that information?

File: CaseStudy.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+#### Key Features
+- A no-code low-code solution which can be quickly setup to automate data collection on edge.
+- Supports intermittent internet connectivity and provides an asynchronous way for data upload from edge.
+- Support upload of high-quality large size images for model training on the cloud
+
+### Deployment Size Optimizations for Constrained Network
+To address key challenges around heavy edge images leading to exponential deployments (caused by multiple failures), lack of support for limited network bandwidth and high network cost, a two-stage approach was implemented.
+#### <ins>1.	Orchestrated Inferencing :</ins> The concept of Orchestrated inferencing was introduced which dictate the design based on decoupling the ML models and orchestration logic into separate docker containers. 
+
+ ![image](/docs/images/Orchestrated_Inferencing.png)
```
Comment (ritesh-modi) [inline CaseStudy.md:101]: nit: the width and height of this image can be reduced.

**Review by ritesh-modi — CHANGES_REQUESTED** [review]
LGTM apart from a small comment.

File: CaseStudy.md (hunk 8 above)
Comment (saritab07) [reply CaseStudy.md:101]: Tried changing the width and the height of the image. Not sure if this is quite readable. Please review once.

**Review by ritesh-modi — APPROVED** [review]
The image size is still quite large. It would be good to resize it.

**Pull request merged**